src/drivers/uart/src/debug_port.rs

```diff
@@ -20,8 +20,8 @@ impl<D: Driver> DebugPort<D> {
 }
 
 impl<D: Driver> Driver for DebugPort<D> {
-    // Nothing to set up here
     fn init(&mut self) -> Result<()> {
+        self.timer.enable().unwrap();
```

Comment:
Probably makes more sense to `map_err` here. `unwrap` would just fail the program on error.